### ISSUE TITLE
rstudio server didn't use user's R library path first when resuming from a session

### ISSUE THREAD

**Issue by dracodoc** [rstudio/rstudio#5819]
Our company allow users to install their own library to user library path in RStudio server (Pro 1.1.453).
When resuming from a suspended session, RStudio server loaded the older version of Rcpp (0.12.19) installed in server R library first(not using the higher version installed in user's R library, which is in the first of libpath), then loaded many packages until met error with dplyr:

> Error: package or namespace load failed for ‘dplyr’ in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]):
>  namespace ‘Rcpp’ 0.12.19 is already loaded, but >= 1.0.1 is required
> Error restoring session data (loading package dplyr): R code execution error

I tried to detach/unload Rcpp 0.12.19, but it was loaded before many dependent packages, unloading each package require proper order, it became a very tedious task and may still fail. In the end I have to restart session, which lost the saved session.

In summary, this behavior make resuming suspended session useless if user need to use the newer version Rcpp in user R library.

Can RStudio pro check the user Rprofile and load the right version first? I have user R library in first of my search path, but I'm afraid RStudio didn't check user's Rprofile in early stage.

**Comment by jmcphers** [comment]
As this is an issue with RStudio Pro, please open a ticket with our support team (support@rstudio.com). They can likely show you the right place to set this up!
